pkgs/development/libraries/libxml2/default.nix

```diff
@@ -28,34 +43,33 @@ stdenv.mkDerivation rec {
     #   https://github.com/NixOS/nixpkgs/pull/63174
     #   https://github.com/NixOS/nixpkgs/pull/72342
     ./utf8-xmlErrorFuncHandler.patch
```

Comment:
This patch might potentially not be needed any more. But we should probably focus on getting the security fixes in first.